src/gui/virtualstudio.cpp

```diff
@@ -113,9 +113,21 @@ VirtualStudio::VirtualStudio(bool firstRun, QObject* parent)
     m_inMuted       = settings.value(QStringLiteral("InMuted"), false).toBool();
     m_outMuted      = settings.value(QStringLiteral("OutMuted"), false).toBool();
 #ifdef RT_AUDIO
-    m_useRtAudio     = settings.value(QStringLiteral("Backend"), 1).toInt() == 1;
-    m_inputDevice    = settings.value(QStringLiteral("InputDevice"), "").toString();
-    m_outputDevice   = settings.value(QStringLiteral("OutputDevice"), "").toString();
+    m_useRtAudio   = settings.value(QStringLiteral("Backend"), 1).toInt() == 1;
+    m_inputDevice  = settings.value(QStringLiteral("InputDevice"), "").toString();
+    m_outputDevice = settings.value(QStringLiteral("OutputDevice"), "").toString();
+
+    if (m_inputDevice == QStringLiteral("(default)")) {
+        m_inputDevice = "";
+    }
+    if (m_outputDevice == QStringLiteral("(default)")) {
+        m_outputDevice = "";
+    }
+
+    m_baseInputChannel = settings.value(QStringLiteral("BaseInputChannel"), 1).toInt();
+    m_numInputChannels = settings.value(QStringLiteral("NumInputChannels"), 1).toInt();
+    m_inputMixMode = settings.value(QStringLiteral("InputMixMode"), "Mono").toString();
```

Comment:
Does case matter here? `Mono` vs `mono` - I see `m_inputMixMode` being compared to the lower-case "mono" in other parts of the code

Reply:
The mix mode should be an enum (class). Maybe it could be added to AudioInterfaceMode.h

Reply:
Yes this should be lowercase. I probably will go ahead and make this an enum though